api/response/schema.py

```diff
@@ -251,6 +277,45 @@ def mutate(self, info, responses, resolved):
         )
 
 
+class ArchiveResponse(graphene.Mutation):
+    """
+    Returns payload after marking a resolved response
+    as archived
```

Comment:
This docstring does not match this mutation

Reply:
Done